src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -232,7 +244,7 @@ export default {
 
 		toggleCollapsed(messages) {
 			this.$set(messages, 'collapsed', !messages.collapsed)
-			this.groupIsCollapsed[messages.id] = !this.groupIsCollapsed[messages.id]
+			this.groupIsCollapsed[messages.id] = messages.collapsed
```

Comment:
should be also the opposite value, like above? `!messages.collapsed`

Reply:
Apparently, not at the moment:

![image](https://github.com/nextcloud/spreed/assets/93392545/cae92288-05dd-4a6f-8b88-d59c17f2c8e3)